pycue/opencue/cuebot.py

```diff
@@ -58,6 +58,15 @@
 from opencue_proto import subscription_pb2_grpc
 from opencue_proto import task_pb2
 from opencue_proto import task_pb2_grpc
+# Optional monitoring proto - may not be available in all deployments
+# pylint: disable=ungrouped-imports
```

Comment:
I don't think we should conditionally disable monitoring on the api level. This can lead to environments being configured with monitoring turned on at cuebot and off at pycue. IMO monitoring functions and modules should always be available on pycue, calling monitoring functions pointing to cuebot where monitoring is disabled should return a grpc.Status=Unimplemented. This way toggling monitoring can be done at the Cuebot end without requiring a new version of pycue.